src/components/filterPatterns/modal/OpenFilterButton.tsx

```diff
@@ -33,6 +39,8 @@ export const OpenFilterButton: FC<Props> = ({
   onClick,
   variant = 'md',
   isDisabled = false,
+  paddingX = 0,
```

Comment:
i'd probably restrict it to 0 and whatever you need to pass in listings-web. Not just anything.

Reply:
I am fine with that